.github/workflows/run-tests.yml

```diff
@@ -217,36 +245,41 @@ jobs:
         working-directory: neofs-testcases
 
 ################################################################
-      - name: Get Allure history
-        timeout-minutes: 60
-        uses: actions/checkout@v3
-        if: always()
-        with:
-          ref: gh-pages
-          path: gh-pages
-
-      - name: Allure Report action from marketplace
+      - name: Generate Allure report
         timeout-minutes: 60
         uses: simple-elf/allure-report-action@v1.6
         if: always()
         id: allure-report
         with:
-          keep_reports: 200
+          keep_reports: 100000
           allure_results: allure-results
           allure_report: allure-report
-          gh_pages: gh-pages
           allure_history: allure-history
 
-      - name: Deploy report to Github Pages
-        timeout-minutes: 120
+      - name: Remove testing neofs-cli
         if: always()
-        uses: peaceiris/actions-gh-pages@v3.9.3
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_branch: gh-pages
-          publish_dir: allure-history
-          force_orphan: true
-          enable_jekyll: true
+        run: |
+          make clean
+        working-directory: neofs-node
+
+      - name: Enable stable neofs-cli
+        if: always()
+        run: |
+          echo "$(pwd)" >> $GITHUB_PATH
+        working-directory: neofs-node-stable
+
+      - name: Create wallet
+        if: always()
+        run: |
+          echo "$TEST_RESULTS_WALLET" | base64 -d > wallet.json
+        working-directory: neofs-testcases
+
+      - name: Put allure report to NeoFS
+        if: always()
+        run: |
+          sudo chmod -R a+rw ${GITHUB_WORKSPACE}/allure-report
+          source venv.local-pytest/bin/activate && python ./tools/src/process-allure-reports.py --neofs_node $TEST_RESULTS_NEOFS_NODE --run_id $RUN_ID --cid $TEST_RESULTS_CID --allure_report ${GITHUB_WORKSPACE}/allure-report --wallet wallet.json
```

Comment:
will it be done via a separate GH action in the future?

Reply:
> will it be done via a separate GH action in the future?

Sorry for the late reply.
Running tests will not be done via a separate GH action.

Report generation - yes, it will be done.